sonaranalyzer-dotnet/its/regression-test.ps1

```diff
@@ -185,9 +185,9 @@ try {
     Initialize-OutputFolder
 
     Write-Debug "Installing the import before target file at '${msBuildImportBefore14}'"
-    Copy-Item .\SonarAnalyzer.Testing.ImportBefore.targets -Destination $msBuildImportBefore14 -Recurse -Container
+    Copy-Item .\SonarAnalyzer.Testing.ImportBefore.targets -Destination (New-Item $msBuildImportBefore14 -Type container -Force) -Force -Recurse
```

Comment:
Just adding `-Force` would work and is more readable than explicit creation of folder.

See https://stackoverflow.com/questions/2695504/powershell-2-copy-item-which-creates-a-folder-if-doesnt-exist

Reply:
Doesn't work hence this change.

Reply:
Eh, Powershell...